src/pe-sp-tabs-launchpad/README.md

```diff
@@ -0,0 +1,124 @@
+# Sp Tabs Launchpad
+
+## Description
+
+> Tabs component with options and SASS variables for an easy customisation. It is using a provided API / Script Include and Service Portal Menu Items as data source.
```

Comment:
The ">" can simply be removed here; it is typically used just for content blocks with no copy.